.github/workflows/gh-actios.yaml

```diff
@@ -0,0 +1,32 @@
+name: Kuttl tests
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+  
+jobs:
+  run-e2etest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Prepare environment
+        run: make kind-create-cluster 
+      - name: Install kyverno Operator
```

Comment:
why do we need this step?

Reply:
Yes, should be ok not to have kyverno operator, we are testing kyverno only.